### ISSUE TITLE
 Speeck SDK error in angular

### ISSUE THREAD

**Issue by mekacharan** [microsoft/cognitive-services-speech-sdk-js#249]
<img width="683" alt="error" src="https://user-images.githubusercontent.com/45178062/94799470-cbc30980-0400-11eb-8555-55d1e9c58884.png">
I am working on angular app and trying to integrate speech sdk for voice feature. I ran npm i microsoft-cognitiveservices-speech-sdk in my solution . When i am building the angular app I am getting below error. 
ERROR in ../node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/CustomCommandsConfig.d.ts:45:9 - error TS2611: 'applicationId' is defined as a property in class 'DialogServiceConfigImpl', but is overridden here in 'CustomCommandsConfig' as an accessor.

45     set applicationId(value: string);

Please find attached image for reference

**Comment by ChrisCSmith** [comment]
I am having similar issue:

version 1.13.0 has the following error:
Error: node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/CustomCommandsConfig.d.ts:45:9 - error TS2611: 'applicationId' is defined 
as a property in class 'DialogServiceConfigImpl', but is overridden here in 'CustomCommandsConfig' as an accessor.

45     set applicationId(value: string);

versions 1.14.0 and 1.14.1 have the following errors (bleow) which look similar to the following issue:
https://github.com/microsoft/cognitive-services-speech-sdk-js/issues/166

Error: node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Transcription/ConversationTranslator.d.ts:29:5 - error TS2416: Property 'canceled' in type 'ConversationTranslator' is not assignable to the same property in base type 'IConversationTranslator'.
  Type '(sender: IConversationTranslator, event: ConversationTranslationCanceledEventArgs) => void' is not assignable to type '(sender: ConversationHandler, event: ConversationTranslationCanceledEventArgs) => void'.
    Types of parameters 'sender' and 'sender' are incompatible.
      Type 'ConversationHandler' is missing the following properties from type 'IConversationTranslator': properties, speechRecognitionLanguage, conversationExpiration, participantsChanged, and 5 more.

29     canceled: (sender: IConversationTranslator, event: ConversationTranslationCanceledEventArgs) => void;
       ~~~~~~~~
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Transcription/ConversationTranslator.d.ts:32:5 - error TS2416: Property 'sessionStarted' in type 'ConversationTranslator' is not assignable to the same property in base type 'IConversationTranslator'.
  Type '(sender: IConversationTranslator, event: SessionEventArgs) => void' is not assignable to type '(sender: ConversationHandler, event: SessionEventArgs) => void'.
    Types of parameters 'sender' and 'sender' are incompatible.
      Type 'ConversationHandler' is not assignable to type 'IConversationTranslator'.

32     sessionStarted: (sender: IConversationTranslator, event: SessionEventArgs) => void;
m       ~~~~~~~~~~~~~~
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Transcription/ConversationTranslator.d.ts:33:5 - error TS2416: Property 'sessionStopped' in type 'ConversationTranslator' is not assignable to the same property in base type 'IConversationTranslator'.
  Type '(sender: IConversationTranslator, event: SessionEventArgs) => void' is not assignable to type '(sender: ConversationHandler, event: SessionEventArgs) => void'.
    Types of parameters 'sender' and 'sender' are incompatible.
      Type 'ConversationHandler' is not assignable to type 'IConversationTranslator'.

33     sessionStopped: (sender: IConversationTranslator, event: SessionEventArgs) => void;
       ~~~~~~~~~~~~~~
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Transcription/ConversationTranscriber.d.ts:21:5 - error TS2416: Property 'canceled' in type 'ConversationTranscriber' is not assignable to the same property in base type 'ConversationTranscriptionHandler'.
  Type '(sender: ConversationTranscriptionHandler, event: CancellationEventArgs) => void' is not assignable to type '(sender: ConversationHandler, event: ConversationTranslationCanceledEventArgs) => void'.
    Types of parameters 'sender' and 'sender' are incompatible.
      Type 'ConversationHandler' is missing the following properties from type 'ConversationTranscriptionHandler': transcribed, transcribing, joinConversationAsync

21     canceled: (sender: ConversationTranscriptionHandler, event: CancellationEventArgs) => void;
       ~~~~~~~~
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Transcription/ConversationTranscriber.d.ts:46:5 - error TS2416: Property 'sessionStarted' in type 'ConversationTranscriber' is not assignable to the same property in base type 'ConversationTranscriptionHandler'.
  Type '(sender: ConversationTranscriptionHandler, event: SessionEventArgs) => void' is not assignable to type '(sender: ConversationHandler, event: SessionEventArgs) => void'.
    Types of parameters 'sender' and 'sender' are incompatible.
      Type 'ConversationHandler' is not assignable to type 'ConversationTranscriptionHandler'.

46     sessionStarted: (sender: ConversationTranscriptionHandler, event: SessionEventArgs) => void;
       ~~~~~~~~~~~~~~
node_modules/microsoft-cognitiveservices-speech-sdk/distrib/lib/src/sdk/Transcription/ConversationTranscriber.d.ts:53:5 - error TS2416: Property 'sessionStopped' in type 'ConversationTranscriber' is not assignable to the same property in base type 'ConversationTranscriptionHandler'.
  Type '(sender: ConversationTranscriptionHandler, event: SessionEventArgs) => void' is not assignable to type '(sender: ConversationHandler, event: SessionEventArgs) => void'.
    Types of parameters 'sender' and 'sender' are incompatible.
      Type 'ConversationHandler' is not assignable to type 'ConversationTranscriptionHandler'.

53     sessionStopped: (sender: ConversationTranscriptionHandler, event: SessionEventArgs) => void;
       ~~~~~~~~~~~~~~0m

**Comment by prayatna** [comment]
same issue as @ChrisCSmith

**Comment by glharper** [comment]
@mekacharan I believe the original issue was fixed in 1.14.1 with [this PR](https://github.com/microsoft/cognitive-services-speech-sdk-js/pull/236). If this is still happening in our latest release (1.17), please reopen and show the error as before. Thanks!
